Assets/Scripts/GameManager.cs

```diff
@@ -68,15 +85,47 @@ public class GameManager : MonoBehaviour {
 
     private void FillUITexts() {
         if (!currentTextAlreadyFilled) {
+            Statement CurrQuestion = CurrentQuestion;
```

Comment:
This is useless, you're just copying the reference of CurrentQuestion to CurrQuestion, they both point to the same object